### PR TITLE
sort apps, set tag and repo

### PR DESCRIPTION
**Pull request by quinn**
```
$ tuber apps set-branch [app] [tag]
$ tuber apps set-repo [app] [repo]
```

### CODE REVIEW

File: cmd/apps.go [hunk 1]
```diff
@@ -27,6 +28,44 @@ var appsInstallCmd = &cobra.Command{
...
+			return err
+		}
+
+		return core.AddAppConfig(appName, app.Repo, tag)
+	},
+}
+
+var appsSetRepoCmd = &cobra.Command{
+	SilenceUsage: true,
+	Use:          "[app name] [docker repo]",
```
Comment (Jefferson-Faseler) [inline cmd/apps.go]: This doesn't make the command available

I think it needs `set-repo` in front

**Comment by Jefferson-Faseler** [comment]
@quinn what's the latest on this?

File: cmd/apps.go [hunk 2]
```diff
@@ -66,6 +104,8 @@ var appsListCmd = &cobra.Command{
 		table.SetHeader([]string{"Name", "Image"})
 		table.SetBorder(false)
 
+		sort.Sort(apps)
```
Comment (jkminneti) [inline cmd/apps.go]: seems like `sort.Slice` would make this a lot easier

Comment (quinn) [reply cmd/apps.go]: i don't understand this feedabck

Comment (jkminneti) [reply cmd/apps.go]: https://github.com/Freshly/tuber/pull/93

**Pull request merged**